fix: push docker manifest to combine arm and amd

cheyer opened the pull request.

velramiir commented on .github/workflows/build-docker-image-for-arch.yml:
```diff
@@ -0,0 +1,81 @@
+          FONTAWESOME_NPM_AUTH_TOKEN: ${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}
+          PLATFORM: ${{ fromJson(steps.configure-platform.outputs.result) }}
+        run: |
+          # copose image ID and change all uppercase to lowercase
```
```suggestion
          # compose image ID and change all uppercase to lowercase
```

velramiir reviewed (commented):
Love this :heart: 

Since the arm images now build as fast as the amd ones, I'm wondering if we should reduce the complexity and in this the probability for errors by getting rid of the option to build only for one architecture. Building for both could become the one and only option.

Also, while you're touching this, what do you think about updating all the action versions, so the warning about Node 20 goes away?

cheyer commented:
> Love this ❤️
> 
> Since the arm images now build as fast as the amd ones, I'm wondering if we should reduce the complexity and in this the probability for errors by getting rid of the option to build only for one architecture. Building for both could become the one and only option.
> 
> Also, while you're touching this, what do you think about updating all the action versions, so the warning about Node 20 goes away?

I've updated all corresponding repos that use this action and removed the architecture option.
For the node 20 warning I would propose a seperate PR, not to mix concerns.
